cli-feed-v3-3.json

```diff
@@ -0,0 +1,317 @@
+{
+  "tags": {
+    "v1": {
+      "release": "1.13.0",
+      "displayName": ".NET Framework",
+      "displayVersion": "v1",
+      "releaseQuality": "GA",
+      "targetFramework": ".NET Framework",
+      "capabilities": "",
```

Comment:
Added `"capabilites"` that we can use in future. Tools would be expected to only iterate over tags/trains that they have the capability for.